condor.go

```diff
@@ -105,7 +105,8 @@ func New(c *viper.Viper, client Messenger, fs fsys) *CondorLauncher {
 func (cl *CondorLauncher) GenerateCondorSubmit(submission *model.Job) (string, error) {
 	tmpl := `universe = vanilla
 executable = /usr/local/bin/road-runner
-rank = mips
+rank = mips{{ if .UsesVolumes }}
```

Comment:
as an aside: we might want to change this `rank` expression at some point, because it's probably what's causing some nodes to get used all the time and others to be basically ignored. I'm not sure what's better and it's definitely out of scope for this PR though!